theme/lumo/vaadin-details-styles.html

```diff
@@ -10,23 +10,22 @@
     <style>
       :host {
         margin: var(--lumo-space-xs) 0;
-        font-family: var(--lumo-font-family);
```

Comment:
I thought we should not touch the user-provided content inside the component, let the font family inherit from the parent. We should ideally have some guidelines about this, I think it’s currently a little mixed between components.